extras/lbf-nix/build.nix

```diff
@@ -48,5 +48,18 @@
         lbfPlutusRust = import ./lbf-plutus-rust.nix pkgs config.packages.lbf lbg-rust;
       };
 
+      packages = {
+        lbf-list-modules-typescript = pkgs.stdenv.mkDerivation {
+          name = "lbf-list-modules-typescript";
+          buildInputs = [ pkgs.nodejs ];
+          dontUnpack = true;
+          src = ./lbf-list-modules-typescript;
+          installPhase = ''
+            mkdir -p "$out/bin"
+            cp "$src" "$out/bin/lbf-list-modules-typescript"
+          '';
+        };
+      };
+
```

Comment:
What's going on here?

Recall in the TypeScript code generator, when we import a LB module, we need a bit more information like the package name
```ts
import * as Blargh from '<package-name>/Module/Name.mjs
```

Getting the JSON object which maps package names to LB module names used to be magic nix code -- this derivation allows one to call that magic code themselves (and indeed, has been reimplemented in JS)

Reply:
> I struggle to understand why did you separate this out? The LB package names are specified by (Nix) users, so the entire information is provided by Nix users and the Nix machinery can do everything under the hood.

Yes, the old workflow was: nix provides info ---> code generator.
Now, this still allows the old workflow, but ALSO allows: user provides info from cli (via this tool) ---> code generator.

In a sense, this allows non-nix users to still generate types.

> The command line interface users also MUST specify the package name, and that's what this tool does?

Yes, the LB code generator must have a mapping from NPM package names to LambdaBuffers' modules. This tool helps provide this mapping which used to only exist on nix.

This was done to resolve the `import` problem described above.